resharper/resharper-unity/src/Application/UI/Help/ShowUnityHelp.cs

```diff
@@ -37,25 +36,39 @@ public bool ShowHelp(string keyword, HelpSystem.HelpKind kind, string preferredP
         public double Priority => 0;
         public HelpSystem.HelpKind[] SupportedKinds => new[] {HelpSystem.HelpKind.Msdn};
 
-        private static bool IsUnityKeyword(string keyword)
+        public static bool IsUnityKeyword(string keyword)
         {
-            return keyword.StartsWith("UnityEngine.", StringComparison.OrdinalIgnoreCase)
-                   || keyword.StartsWith("UnityEditor.", StringComparison.OrdinalIgnoreCase);
+            return keyword !=null && 
```

Comment:
What does it mean to be called with `null`? What help is ReSharper trying to show in that case? And if you've got the chance, can you add `[CanBeNull]` to the interface?